iree/schemas/BUILD

```diff
@@ -33,7 +34,16 @@ FLATC_ARGS = [
     "--gen-object-api",
 ]
 
-# TODO(benvanik): also expose as C using flatcc.
+iree_flatbuffer_c_library(
+    name = "bytecode_module_def_c_fbs",
+    srcs = ["bytecode_module_def.fbs"],
+    flatcc_args = [
+        "--reader",
+        "--builder",
+        "--verifier",
+    ],
+)
```

Comment:
Integration on the Google side may be easier if we stage this PR into two changes:
* a first PR that includes all the preceding commits (before "Building bytecode_module_def with flatcc.") and this (and other) usage commented out
* a second PR that flips this to be included

Reply:
have reset this to point prior to building bytecode_module_def